vitrina/classifiers/migrations/0015_distribution_status_concept_schema.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.25 on 2025-08-26 13:01
+
+from django.db import migrations
+
+# Generated by Django 3.2.25 on 2025-08-19 11:43
+
+from django.db import migrations
+
+URI="http://publications.europa.eu/resource/authority/distribution-status"
+
+
+def create_concept_schema(apps, schema_editor):
+    concept_schema = apps.get_model("vitrina_classifiers", "ConceptSchema")
+    schema, created = concept_schema.objects.get_or_create(
```

Comment:
```suggestion
    ConceptSchema = apps.get_model("vitrina_classifiers", "ConceptSchema")
    schema, created = ConceptSchema.objects.get_or_create(
```

Convention to use native model name case

Reply:
Renamed